src/jukebox/components/rfid/hardware/generic_nfcpy/generic_nfcpy.py

```diff
@@ -0,0 +1,90 @@
+# Standard imports from python packages
+import logging
+
+import nfc
+from nfc.clf import RemoteTarget
+
+# Import the ReaderBaseClass for common API. Leave as this line as it is!
+from components.rfid import ReaderBaseClass
+import jukebox.cfghandler
+
+# Also import the description string into this module, to make everything available in a single module w/o code duplication
+# Leave this line as is!
+from .description import DESCRIPTION
+
+# Create logger.
+# Logging is fully setup. Just replace '.new' with something meaningful and short
+logger = logging.getLogger('jb.rfid.new')
+# Get the global handler to the RFID config
+cfg = jukebox.cfghandler.get_handler('rfid')
+
+
+
+
+def query_customization() -> dict:
+    """
+    Query the user for reader parameter customization
```

Comment:
I think at least the DRIVER parameter should be queried here.

see https://github.com/MiczFlor/RPi-Jukebox-RFID/tree/future3/develop/src/jukebox/components/rfid/hardware/rc522_spi for an example